system/nagios-status-check/detectors-nagios-status-check.tf

```diff
@@ -0,0 +1,27 @@
+resource "signalfx_detector" "status_check" {
```

Comment:
could be done in another future PR but could be useful to also add an heartbeat detector.

In this case, I am pretty sure we will have to cover both cases (erratic or regular reporting) so we could eventually add a boolean flag parameter to the wrapper script and add a dimension "regular" or wathever which will allow to filter only on those checks from the hearbeat detector.

From this way:
- we could be alerted of not reporting checks which should always do
- but allow the manage the exception for erratic checks just by adding the right parameter to the wrapper script in the monitor conf